boltdesignsystem/bolt
# DS-170: spacing hotfix for Japanese

## Jira

https://pegadigitalit.atlassian.net/browse/DS-170

## Summary

Fixes an issue where the new CSS spacing vars are not producing the same values as the old spacing mixins and functions in Japanese build.

## Details

1. Added a new file under global/generic
2. CSS vars are redefined and scoped to `[lang='ja']`

## How to test

In Drupal staging environment, use language switcher to switch to Japanese. Make sure layouts are still the same as before, nothing is misaligned.
